src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,121 @@
+"""
+
+estimate.py
+
+Functions to help with estimation of low-light parameters.
+
+"""
+
+import sys
```

Comment:
Supposedly this isn't needed if you ran the command:
```bash
python -m pip install ./sipmpdf
python -m pip install ./sipmanalyze
```
During the instruction, as indicate in the README file [1]. Let me know this doesn't work out. 

[1] https://github.com/UMDCMS/sipmanalyze#setup